docs/integrations/destinations/snowflake.md

```diff
@@ -189,6 +189,7 @@ Finally, you need to add read/write permissions to your bucket with that email.
 
 | Version | Date      | Pull Request | Subject |
```

Comment:
@dsdorazio would you mind adding a section above (similar to the GCS one) for Azure? 

I set one up for testing. here is what I ran on snowflake, for reference: 

```
# Create an integration between snowflake and azure
create storage INTEGRATION azure_airbyte_integration
  TYPE = EXTERNAL_STAGE
  STORAGE_PROVIDER = AZURE
  ENABLED = TRUE
  AZURE_TENANT_ID = '<azure_tenant_id>'
  STORAGE_ALLOWED_LOCATIONS = ('azure://<account_name>.blob.core.windows.net/<blob_container>/');

# Create a stage
create stage azure_airbyte_stage
  url = '<storage_allowed_location_url_above>'
  storage_integration = azure_airbyte_integration;
GRANT USAGE ON integration azure_airbyte_integration TO ROLE AIRBYTE_ROLE;
GRANT USAGE ON stage azure_airbyte_stage TO ROLE AIRBYTE_ROLE;
DESC STORAGE INTEGRATION azure_airbyte_integration;
```